src/test/java/kitchenpos/application/InMemoryOrderRepository.java

```diff
@@ -0,0 +1,41 @@
+package kitchenpos.application;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import kitchenpos.domain.Order;
+import kitchenpos.domain.OrderRepository;
+import kitchenpos.domain.OrderStatus;
+import kitchenpos.domain.OrderTable;
+
+public class InMemoryOrderRepository implements OrderRepository {
+
+    private final Map<UUID, Order> orders = new HashMap<>();
+
+    @Override
+    public Optional<Order> findById(UUID orderId) {
+        return Optional.ofNullable(orders.get(orderId));
+    }
+
+    @Override
+    public boolean existsByOrderTableAndStatusNot(OrderTable orderTable, OrderStatus status) {
+        return orders.values().stream()
+            .anyMatch(
+                order -> order.getOrderTableId().equals(orderTable.getId()) && !order.getStatus().equals(status));
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
                order -> order.getOrderTableId().equals(orderTable.getId()) && order.getStatus() != status);
```